.github/workflows/job-postcommit-beam-metrics-publish.yml

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+# Applies new configuration to Beam Metrics infrastructure.
+
+name: PostCommit Beam Metrics Publish
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master']
+permissions: read-all
+
+jobs:
+  postcommit-beam-metrics-publish:
+    name: Run PostCommit Beam Metrics Publish
+    runs-on: [self-hosted, ubuntu-20.04]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          submodules: recursive
```

Comment:
Do we really need to check out the submodules here? I imagine for most of these workflows we don't. The only places we should need them are when we're using one of the actions listed here (https://github.com/apache/beam/blob/master/.gitmodules) which I don't think we do in any of the migrated workflows, or when updating the website (where it may or may not be necessary)